src/main/resources/assets/tconstruct/book/encyclopedia/zh_tw/abilities/armor.json

```
@@ -0,0 +1,11 @@
+{
+  "title": "盔甲能力",
+  "text": [
+    { "text": "•一般來說，盔甲能力只對某一特定欄位的盔甲起作用。不同部位的盔甲效果也有所不同：" },
+    { "text": "\n• 頭盔可以通過綁定按鍵觸發特殊效果。", "paragraph": true },
```

繁中這邊的 `通過`，意思是穿越某個地方，也就是 `Pass through`
其他地方的 `通過` 也需要替換
```suggestion
    { "text": "\n• 頭盔可以透過綁定按鍵觸發特殊效果。", "paragraph": true },
```